server/templates/staff/student/submit.html

```diff
@@ -1,7 +1,7 @@
 {% extends "staff/base.html" %}
 {% import 'staff/_formhelpers.html' as forms with context %}
 
-{% block title %} Ok - New Submission for {{ student.identifier }}{% endblock %}
+{% block title %} Submit Assignment - {{ current_course.display_name_with_semester }}{% endblock %}
```

Comment:
instead of submit for {{ course.display_name }} - we might want to keep the student.identifier here.